run_xnat2bids.py

```diff
@@ -0,0 +1,57 @@
+from getpass import getpass
+import glob
+import os
+import shlex
+import shutil
+import subprocess
+from toml import load
+
+# Load config file into dictionary
+arglist = load('x2b_config.toml')
```

Comment:
Have this be a default config with full-path and load an user config that you overwrite with `dict.update()`